web/viewer.html

```diff
@@ -275,6 +301,8 @@
                   </button>
                 </div>
 
+                <div class="verticalToolbarSeparator hiddenSmallView"></div>
+
```

Comment:
This is adding an *unconditional* separator, which will look weird and out-of-place when editing is disabled (which is still the default state).
Hence this one probably needs to be moved into the `editorModeButtons`-div just above.